### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by klinger**
- You need to build ceylon-dist first - OR you will have to clone many more repositories first.
- repaired enumeration
- better end of chapter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,44 +16,46 @@ _This is the **simplest way** to install the Ceyon IDE Eclipse plugin._
 
 _If you want to have an up-to-date version of the Ceylon IDE based on the lastest code of all Ceylon projects, this is the **prefered method**._ 
 
-1.  First make sure that your Eclipse can be run by simply typing the `eclipse` command (either by adding the `eclipse` command full path to the PATH environment variable, or by creating a symbolic link to the `eclipse` executable file in a directory already visible in the PATH).
+1.  Built the command line distribution first (decribed at https://github.com/ceylon/ceylon-dist ), ant setup prepares the environment and clones addition repositories.
```
Comment (quintesse) [inline README.md]: "Build"

**Comment by klinger** [comment]
thx. repaired.

File: README.md [hunk 2]
```diff
@@ -16,44 +16,46 @@ _This is the **simplest way** to install the Ceyon IDE Eclipse plugin._
 
 _If you want to have an up-to-date version of the Ceylon IDE based on the lastest code of all Ceylon projects, this is the **prefered method**._ 
 
-1.  First make sure that your Eclipse can be run by simply typing the `eclipse` command (either by adding the `eclipse` command full path to the PATH environment variable, or by creating a symbolic link to the `eclipse` executable file in a directory already visible in the PATH).
+1.  Build the command line distribution first (decribed at https://github.com/ceylon/ceylon-dist ), ant setup prepares the environment and clones addition repositories.
```
Comment (pthariensflame) [inline README.md]: "addition" should be "additional"

Comment (pthariensflame) [inline README.md]: Also, `ant setup` should be inside backticks.

File: README.md [hunk 3]
```diff
@@ -16,44 +16,46 @@ _This is the **simplest way** to install the Ceyon IDE Eclipse plugin._
...
 
-1b. Make sure that your ant (latest version) and maven (Version in Range from 3.0.5 to 3.2.1) can also be run on the command line.
+2.  Make sure that your Eclipse can be run by simply typing the `eclipse` command (either by adding the `eclipse` command full path to the PATH environment variable, or by creating a symbolic link to the `eclipse` executable file in a directory already visible in the PATH).
 
-2.  Make sure that your JAVA_HOME is set to the right JDK 7 installation
+3. Make sure that your ant (latest version) and maven (Version in Range from 3.0.5 to 3.2.1) can also be run on the command line.
```
Comment (pthariensflame) [inline README.md]: These parentheticals need some serious rewording in order to be proper English.

File: README.md [hunk 4]
```diff
@@ -16,44 +16,46 @@ _This is the **simplest way** to install the Ceyon IDE Eclipse plugin._
...
 
-    To skip tests completely you can do:
+    Faster alternative: To skip tests completely you can do:
 
         `mvn clean install -DskipTests` 
    
-7.  The directory `site/target/repository` now contains an update site you can 
-    install from.
+10.  The directory `site/target/repository` now contains an update site you can 
+     install from (documentation at http://ceylon-lang.org/documentation/1.0/ide/install/ but use the path to this         directory instead of the stable web repository)
```
Comment (pthariensflame) [inline README.md]: I am boldly assuming you didn't mean to put a tab character in the middle of a line here.  :P

**Comment by davidfestal** [comment]
finally ! sorry for merging this so late !

**Pull request merged**